ubuntudesign/canonical-vanilla-theme
# Forms

Done:
A couple of simple overrides for canonical.com style markup with vanilla forms.

QA:
(bleeding edge vanilla) gulp build
view demo/index.html

Ensure that the form element at the bottom of the document has reasonable layout and no bullets. (Design will be consulted for any further tweaks)
